src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -155,11 +153,8 @@ class Liberty implements Plugin<Project> {
             description 'Starts the WebSphere Liberty Profile server.'
             logging.level = LogLevel.INFO
             group 'Liberty'
-            dependsOn 'libertyCreate'
 
-            project.afterEvaluate {
-                if (dependsOnApps(server)) dependsOn 'installApps'
-            }
+            project.afterEvaluate { dependsOn installDependsOn(server, 'libertyCreate') }
```

Comment:
Feels like a regression here.  David recently made changes here and moved the `dependsOn 'libertyCreate'` outside of the `afterEvaluate`.

Also, recheck the `libertyStart` project.task definition.

Reply:
I manually brought in these changes from @mattbsox 's fork. I don't need them for this PR and can remove the lberty.groovy file from this PR.